CMakeLists.txt

```diff
@@ -21,6 +21,11 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
+# Enable MSVC runtime library setting 
+# (https://cmake.org/cmake/help/latest/prop_tgt/MSVC_RUNTIME_LIBRARY.html)
+cmake_policy(SET CMP0091 NEW) 
```

Comment:
Btw. in contrast to other policies this one default to the old behaviour, disabling the MSVC_RUNTIME_LIBRARY flag.

"This policy was introduced in CMake version 3.15. Use the cmake_policy() command to set it to OLD or NEW explicitly. Unlike many policies, CMake version 3.20.20210503-gcad75bc does not warn when this policy is not set and simply uses OLD behavior."
https://cmake.org/cmake/help/git-stage/policy/CMP0091.html